### ISSUE TITLE
Trailing comma in object throws SyntaxError

### ISSUE THREAD

**Issue by joepie91** [dinoboff/firebase-json#6]
Testcase:

```json
{
	"foo": "bar",
}
```

Error:

```
SyntaxError: Line 3, column 1: Expected "\"" but "}" found.
    at error (/home/sven/projects/xxx/node_modules/firebase-json/index.js:26:15)
    at Object.exports.ast (/home/sven/projects/xxx/node_modules/firebase-json/index.js:83:11)
    at Object.exports.parse (/home/sven/projects/xxx/node_modules/firebase-json/index.js:95:23)
    at Object.<anonymous> (/home/sven/projects/xxx/scripts/generate-migration-paths.js:9:26)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
```

Original case can be found [here](https://github.com/webhook/webhook-server-open/blob/master/security.json) (line 67, column 23), and Firebase definitely accepts this as valid "JSON".

**Comment by dinoboff** [comment]
Thanks. Will fix.

**Comment by dinoboff** [comment]
Firebase arrays (for ".indexOn" rules) also accept trailing commas. It should be fixed in upcoming v0.4.0
